eng/helix/content/runtests.sh

```diff
@@ -103,10 +103,14 @@ fi
 
 FLAKY_FILTER="Flaky:All=true|Flaky:Helix:All=true|Flaky:Helix:Queue:All=true|Flaky:Helix:Queue:$helix_queue_name=true"
 echo "Running known-flaky tests."
-$DOTNET_ROOT/dotnet vstest $test_binary_path --TestCaseFilter:"$FLAKY_FILTER"
+$DOTNET_ROOT/dotnet vstest $test_binary_path --logger:trx --TestCaseFilter:"$FLAKY_FILTER"
```

Comment:
Where do the TRX files end up? I wonder because only *.log files are copied into the magic `$HELIX_WORKITEM_UPLOAD_ROOT` directory or its parent.

Reply:
Trx files are automatically uploaded by the helix infrastructure to azdo, this is how they show up in the test rules tabs, so they come for 'free' to us

Reply:
Backing out the trx on flaky run since its causing the jobs to fail